### ISSUE TITLE
backend: delanswerandfollow does not flag adjascent answers as deleted

### ISSUE THREAD

**Issue by RoboSparrow** [gardners/surveysystem#216]
consider session

```csv
delanswerandfollowing/0b39e0c93f1b338433b8a8773d57f738dd405e17
question1:Answer 1:0:0:0:0:0:0:0::0
question2:Answer 2:0:0:0:0:0:0:0::0
question3:Answer 3:0:0:0:0:0:0:0::0
```

running `request 200 delanswerandfollowing?sessionid=$SESSION&questionid=question1`

expected:

```csv
delanswerandfollowing/0b39e0c93f1b338433b8a8773d57f738dd405e17
question1:Answer 1:0:0:0:0:0:0:0::1
question2:Answer 2:0:0:0:0:0:0:0::1
question3:Answer 3:0:0:0:0:0:0:0::1
```
retrieved

```csv
delanswerandfollowing/0b39e0c93f1b338433b8a8773d57f738dd405e17
question1:Answer 1:0:0:0:0:0:0:0::1
question2:Answer 2:0:0:0:0:0:0:0::0
question3:Answer 3:0:0:0:0:0:0:0::0
```
Another issue is, that the **test passes** !

I supply the test with a temporary fix for #198 in a new PR

**Comment by RoboSparrow** [comment]
PR #218

**Comment by RoboSparrow** [comment]
resolved in pending PR #218
